include/nodejs/node_osrm_support.hpp

```diff
@@ -96,6 +97,18 @@ inline void ParseResult(const osrm::Status &result_status, osrm::json::Object &r
 }
 
 inline void ParseResult(const osrm::Status & /*result_status*/, const std::string & /*unused*/) {}
+inline void ParseResult(const osrm::Status &result_status,
+                        const flatbuffers::FlatBufferBuilder &fbs_builder)
+{
+    auto fbs_result = osrm::engine::api::fbresult::GetFBResult(fbs_builder.GetBufferPointer());
```

Comment:
Done using [ParseResult for JSON](https://github.com/Project-OSRM/osrm-backend/blob/d74e7b66bdc5e9101ac3f1f74b969f5b6c84756a/include/nodejs/node_osrm_support.hpp#L78) as a reference. We also erase `message` and `code` there, but it is not so easy for flatbuffers(it is kind of readonly) and tbh I don't get why are we doing that at all.